<head>
<title>MetNetComp/CoreGenes</title>
<script async src="https://www.googletagmanager.com/gtag/js?id=G-014FPN14MZ"></script>
 <script> window.dataLayer = window.dataLayer || []; function
 gtag(){dataLayer.push(arguments);} gtag('js', new Date());
 gtag('config', 'G-014FPN14MZ');</script>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database [1] / Core Genes</font></b><br><br>
A set of core genes for simulation-based growth-coupled production. You can also see <a href=e_coli_core&for_e&1.html>minimal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/e_coli_core">e_coli_core</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/e_coli_core/metabolites/for_e">for_e</a><br>
Core genes for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp #Remaining genes : 41 <br>
<font color="green">&nbsp Remaining genes: 
<a href=https://www.genome.jp/entry/eco:s0001>s0001</a> <a href=https://www.genome.jp/entry/eco:b0118>b0118</a> <a href=https://www.genome.jp/entry/eco:b3734>b3734</a> <a href=https://www.genome.jp/entry/eco:b3733>b3733</a> <a href=https://www.genome.jp/entry/eco:b3736>b3736</a> <a href=https://www.genome.jp/entry/eco:b3737>b3737</a> <a href=https://www.genome.jp/entry/eco:b3738>b3738</a> <a href=https://www.genome.jp/entry/eco:b3735>b3735</a> <a href=https://www.genome.jp/entry/eco:b3731>b3731</a> <a href=https://www.genome.jp/entry/eco:b3732>b3732</a> <a href=https://www.genome.jp/entry/eco:b0720>b0720</a> <a href=https://www.genome.jp/entry/eco:b2975>b2975</a> <a href=https://www.genome.jp/entry/eco:b2779>b2779</a> <a href=https://www.genome.jp/entry/eco:b2097>b2097</a> <a href=https://www.genome.jp/entry/eco:b0904>b0904</a> <a href=https://www.genome.jp/entry/eco:b2416>b2416</a> <a href=https://www.genome.jp/entry/eco:b2415>b2415</a> <a href=https://www.genome.jp/entry/eco:b1779>b1779</a> <a href=https://www.genome.jp/entry/eco:b2417>b2417</a> <a href=https://www.genome.jp/entry/eco:b1621>b1621</a> <a href=https://www.genome.jp/entry/eco:b3870>b3870</a> <a href=https://www.genome.jp/entry/eco:b3213>b3213</a> <a href=https://www.genome.jp/entry/eco:b3212>b3212</a> <a href=https://www.genome.jp/entry/eco:b1136>b1136</a> <a href=https://www.genome.jp/entry/eco:b2133>b2133</a> <a href=https://www.genome.jp/entry/eco:b1603>b1603</a> <a href=https://www.genome.jp/entry/eco:b1602>b1602</a> <a href=https://www.genome.jp/entry/eco:b1723>b1723</a> <a href=https://www.genome.jp/entry/eco:b0902>b0902</a> <a href=https://www.genome.jp/entry/eco:b0903>b0903</a> <a href=https://www.genome.jp/entry/eco:b4025>b4025</a> <a href=https://www.genome.jp/entry/eco:b2926>b2926</a> <a href=https://www.genome.jp/entry/eco:b0755>b0755</a> <a href=https://www.genome.jp/entry/eco:b2987>b2987</a> <a href=https://www.genome.jp/entry/eco:b3956>b3956</a> <a href=https://www.genome.jp/entry/eco:b1854>b1854</a> <a href=https://www.genome.jp/entry/eco:b4301>b4301</a> <a href=https://www.genome.jp/entry/eco:b4090>b4090</a> <a href=https://www.genome.jp/entry/eco:b2464>b2464</a> <a href=https://www.genome.jp/entry/eco:b2935>b2935</a> <a href=https://www.genome.jp/entry/eco:b3919>b3919</a> &nbsp <a href="../alternativeGenes/alternative_genes&e_coli_core&for_e&maximal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: AddGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/AddGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.128743 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.621403 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 0.702009<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.473606<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 0.230025<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 21.247286<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_lac__D_e">EX_lac__D_e</a> : 17.906977<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 0.844115<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_for_e">EX_for_e</a> : 0.621403<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pyr_e">EX_pyr_e</a> : 0.136326<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/e_coli_core&for_e&core.json" download="../json/e_coli_core&for_e&core.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=e_coli_core.Core%20metabolism&tool=Builder&model=e_coli_core"> Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Orth, J. D., Fleming, R. M., Palsson, B. Ø. (2010). 
Reconstruction and use of microbial metabolic networks: the core Escherichia coli metabolic model as an educational guide. EcoSal plus, 4(1).<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 27-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
