<head>
<title>MetNetComp</title>
<script async src="https://www.googletagmanager.com/gtag/js?id=G-014FPN14MZ"></script>
 <script> window.dataLayer = window.dataLayer || []; function
 gtag(){dataLayer.push(arguments);} gtag('js', new Date());
 gtag('config', 'G-014FPN14MZ');</script>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database [1] / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=e_coli_core&acald_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/e_coli_core">e_coli_core</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/e_coli_core/metabolites/acald_c">acald_c</a><br>
List of minimal gene deletion strategies <a href="../csv/e_coli_core&acald_c.csv" download="e_coli_core&acald_c.csv">(Download)</a><br><br>
Gene deletion strategy (1 of 4: <a href="e_coli_core&acald_c&2.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 10<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:s0001>s0001</a> <a href=https://www.genome.jp/entry/eco:b4077>b4077</a> <a href=https://www.genome.jp/entry/eco:b4015>b4015</a> <a href=https://www.genome.jp/entry/eco:b1479>b1479</a> <a href=https://www.genome.jp/entry/eco:b1603>b1603</a> <a href=https://www.genome.jp/entry/eco:b3962>b3962</a> <a href=https://www.genome.jp/entry/eco:b2297>b2297</a> <a href=https://www.genome.jp/entry/eco:b2458>b2458</a> <a href=https://www.genome.jp/entry/eco:b1676>b1676</a> <a href=https://www.genome.jp/entry/eco:b1854>b1854</a> &nbsp <a href="../alternativeGenes/alternative_genes&e_coli_core&acald_c&1&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.039237 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 4.059754 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 0.213950<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.144340<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 21.655542<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_succ_e">EX_succ_e</a> : 9.704194<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 7.943184<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 4.059754<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_etoh_e">EX_etoh_e</a> : 3.506891<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pyr_e">EX_pyr_e</a> : 1.460065<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/e_coli_core&acald_c&minimal&1.json" download="../json/e_coli_core&acald_c&minimal&1.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=e_coli_core.Core%20metabolism&tool=Builder&model=e_coli_core"> Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Orth, J. D., Fleming, R. M., Palsson, B. Ø. (2010). 
Reconstruction and use of microbial metabolic networks: the core Escherichia coli metabolic model as an educational guide. EcoSal plus, 4(1).<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 27-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
