<head>
<title>MetNetComp</title>
<script async src="https://www.googletagmanager.com/gtag/js?id=G-014FPN14MZ"></script>
 <script> window.dataLayer = window.dataLayer || []; function
 gtag(){dataLayer.push(arguments);} gtag('js', new Date());
 gtag('config', 'G-014FPN14MZ');</script>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iHN637&ile__L_e&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iHN637">iHN637</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iHN637/metabolites/ile__L_e">ile__L_e</a><br>
List of minimal gene deletion strategies <a href="../csv/iHN637&ile__L_e.csv" download="iHN637&ile__L_e.csv">(Download)</a><br><br>
Gene deletion strategy (17 of 24: <a href="iHN637&ile__L_e&18.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 7<br>
<font color="green">&nbsp Gene deletion: 
CLJU_RS11895 CLJU_RS00610 CLJU_RS01155 CLJU_RS06110 CLJU_RS06260 CLJU_RS01595 CLJU_RS05590 &nbsp <a href="../alternativeGenes/alternative_genes&iHN637&ile__L_e&17&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.053966 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.122280 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fru_e">EX_fru_e</a> : 5.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 4.004121<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.158623<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.062527<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.010317<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe2_e">EX_fe2_e</a> : 0.000746<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.000458<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.000275<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.000275<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fol_e">EX_fol_e</a> : 0.000048<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000037<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000037<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ribflv_e">EX_ribflv_e</a> : 0.000024<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000018<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ni2_e">EX_ni2_e</a> : 0.000017<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cobalt2_e">EX_cobalt2_e</a> : 0.000013<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_thm_e">EX_thm_e</a> : 0.000012<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 5.817852<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_etoh_e">EX_etoh_e</a> : 5.033467<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 3.754605<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ser__L_e">EX_ser__L_e</a> : 3.323912<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2_e">EX_h2_e</a> : 2.991070<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 2.569651<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 0.158394<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_met__L_e">EX_met__L_e</a> : 0.148432<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ile__L_e">EX_ile__L_e</a> : 0.122280<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_succ_c">DM_succ_c</a> : 0.006824<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glyc_e">EX_glyc_e</a> : 0.001241<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_mththf_c">DM_mththf_c</a> : 0.000120<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iHN637&ile__L_e&minimal&17.json" download="../json/iHN637&ile__L_e&minimal&17.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/"> Escher site</a> [3].<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Orth, J. D., Fleming, R. M., Palsson, B. Ø. (2010). 
Reconstruction and use of microbial metabolic networks: the core Escherichia coli metabolic model as an educational guide. EcoSal plus, 4(1).<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 27-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
