<head>
<title>MetNetComp</title>
<script async src="https://www.googletagmanager.com/gtag/js?id=G-014FPN14MZ"></script>
 <script> window.dataLayer = window.dataLayer || []; function
 gtag(){dataLayer.push(arguments);} gtag('js', new Date());
 gtag('config', 'G-014FPN14MZ');</script>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database [1] / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iLB1027_lipid&lys__L_h&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iLB1027_lipid">iLB1027_lipid</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iLB1027_lipid/metabolites/lys__L_h">lys__L_h</a><br>
List of minimal gene deletion strategies <a href="../csv/iLB1027_lipid&lys__L_h.csv" download="iLB1027_lipid&lys__L_h.csv">(Download)</a><br><br>
Gene deletion strategy (43 of 46: <a href="iLB1027_lipid&lys__L_h&44.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 27<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/pti:PHATRDRAFT_800>PHATRDRAFT_800</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_42015>PHATRDRAFT_42015</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_54219>PHATRDRAFT_54219</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_22913>PHATRDRAFT_22913</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_55079>PHATRDRAFT_55079</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_draft1186>PHATRDRAFT_draft1186</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_19244>PHATRDRAFT_19244</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_45017>PHATRDRAFT_45017</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_14762>PHATRDRAFT_14762</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_45239>PHATRDRAFT_45239</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_32747>PHATRDRAFT_32747</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_25308>PHATRDRAFT_25308</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_28191>PHATRDRAFT_28191</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_29702>PHATRDRAFT_29702</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_47090>PHATRDRAFT_47090</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_1341>PHATRDRAFT_1341</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_43194>PHATRDRAFT_43194</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_50971>PHATRDRAFT_50971</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_17720>PHATRDRAFT_17720</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_35566>PHATRDRAFT_35566</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_21970>PHATRDRAFT_21970</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_32849>PHATRDRAFT_32849</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_draft1517>PHATRDRAFT_draft1517</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_28585>PHATRDRAFT_28585</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_43697>PHATRDRAFT_43697</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_45428>PHATRDRAFT_45428</a> <a href=https://www.genome.jp/entry/pti:PHATRDRAFT_23365>PHATRDRAFT_23365</a> &nbsp <a href="../alternativeGenes/alternative_genes&iLB1027_lipid&lys__L_h&43&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.253833 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.258840 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_photon_e">EX_photon_e</a> : 250.375232<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 18.898749<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 16.093864<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_no3_e">EX_no3_e</a> : 1.760000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.063029<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.045142<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.004536<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 22.469147<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_for_c">SK_for_c</a> : 4.808728<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 2.674161<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 0.258840<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_biomass_c">DM_biomass_c</a> : 0.253833<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_etoh_e">EX_etoh_e</a> : 0.229553<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_fald_m">DM_fald_m</a> : 0.020411<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iLB1027_lipid&lys__L_h&minimal&43.json" download="../json/iLB1027_lipid&lys__L_h&minimal&43.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/"> Escher site</a> [3].<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 27-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
