<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Wild type metabolic flow</font></b><br><br>
<a href="../indexFiles/index_e_coli_core.html">Search target metabolites</a>.<br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iLJ478"><br>
In wild type, when growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.228407 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 7.349861<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_malt_e">EX_malt_e</a> : 2.794000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 2.368826<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_s_e">EX_s_e</a> : 0.841893<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.175591<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_apoACP_c">SK_apoACP_c</a> : 0.000639<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2_e">EX_h2_e</a> : 17.292389<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 10.382458<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 8.942973<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 8.232495<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2s_e">EX_h2s_e</a> : 0.786423<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_gcald_c">SK_gcald_c</a> : 0.011420<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_5mta_c">SK_5mta_c</a> : 0.001599<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_pap_c">SK_pap_c</a> : 0.000639<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iLJ478&wildtype.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/"> Escher site</a> [4].<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 27-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
