<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database [1] / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iML1515&pyr_e&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iML1515">iML1515</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iML1515/metabolites/pyr_e">pyr_e</a><br>
List of minimal gene deletion strategies <a href="../csv/iML1515&pyr_e.csv" download="iML1515&pyr_e.csv">(Download)</a><br><br>
Gene deletion strategy (104 of 114: <a href="iML1515&pyr_e&105.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 29<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:b1241>b1241</a> <a href=https://www.genome.jp/entry/eco:b0351>b0351</a> <a href=https://www.genome.jp/entry/eco:b2221>b2221</a> <a href=https://www.genome.jp/entry/eco:b0871>b0871</a> <a href=https://www.genome.jp/entry/eco:b2297>b2297</a> <a href=https://www.genome.jp/entry/eco:b2458>b2458</a> <a href=https://www.genome.jp/entry/eco:b2925>b2925</a> <a href=https://www.genome.jp/entry/eco:b2097>b2097</a> <a href=https://www.genome.jp/entry/eco:b2690>b2690</a> <a href=https://www.genome.jp/entry/eco:b3962>b3962</a> <a href=https://www.genome.jp/entry/eco:b1302>b1302</a> <a href=https://www.genome.jp/entry/eco:b2662>b2662</a> <a href=https://www.genome.jp/entry/eco:b4139>b4139</a> <a href=https://www.genome.jp/entry/eco:b4267>b4267</a> <a href=https://www.genome.jp/entry/eco:b4015>b4015</a> <a href=https://www.genome.jp/entry/eco:b0614>b0614</a> <a href=https://www.genome.jp/entry/eco:b0726>b0726</a> <a href=https://www.genome.jp/entry/eco:b3945>b3945</a> <a href=https://www.genome.jp/entry/eco:b4388>b4388</a> <a href=https://www.genome.jp/entry/eco:b2587>b2587</a> <a href=https://www.genome.jp/entry/eco:b4077>b4077</a> <a href=https://www.genome.jp/entry/eco:b0529>b0529</a> <a href=https://www.genome.jp/entry/eco:b1539>b1539</a> <a href=https://www.genome.jp/entry/eco:b0325>b0325</a> <a href=https://www.genome.jp/entry/eco:b0508>b0508</a> <a href=https://www.genome.jp/entry/eco:b4266>b4266</a> <a href=https://www.genome.jp/entry/eco:b4141>b4141</a> <a href=https://www.genome.jp/entry/eco:b1798>b1798</a> <a href=https://www.genome.jp/entry/eco:b3924>b3924</a> &nbsp <a href="../alternativeGenes/alternative_genes&iML1515&pyr_e&104&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.632112 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.174444 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 28.440502<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 6.826753<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.609739<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.159178<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.123384<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe2_e">EX_fe2_e</a> : 0.010152<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.005484<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.003290<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.003290<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000448<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000437<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000216<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ni2_e">EX_ni2_e</a> : 0.000204<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cobalt2_e">EX_cobalt2_e</a> : 0.000016<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 46.568987<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 29.702219<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 7.897122<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 1.914582<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pyr_e">EX_pyr_e</a> : 0.174444<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_5drib_c">DM_5drib_c</a> : 0.000142<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_4crsol_c">DM_4crsol_c</a> : 0.000141<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iML1515&pyr_e&minimal&104.json" download="../json/iML1515&pyr_e&minimal&104.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iJO1366.Central metabolism&tool=Builder&model=iJO1366"> Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 21-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
