<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database [1] / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iML1515&glyclt_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iML1515">iML1515</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iML1515/metabolites/glyclt_c">glyclt_c</a><br>
List of minimal gene deletion strategies <a href="../csv/iML1515&glyclt_c.csv" download="iML1515&glyclt_c.csv">(Download)</a><br><br>
Gene deletion strategy (41 of 83: <a href="iML1515&glyclt_c&42.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 28<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:b4467>b4467</a> <a href=https://www.genome.jp/entry/eco:b1478>b1478</a> <a href=https://www.genome.jp/entry/eco:b1241>b1241</a> <a href=https://www.genome.jp/entry/eco:b4069>b4069</a> <a href=https://www.genome.jp/entry/eco:b3115>b3115</a> <a href=https://www.genome.jp/entry/eco:b1849>b1849</a> <a href=https://www.genome.jp/entry/eco:b2296>b2296</a> <a href=https://www.genome.jp/entry/eco:b1004>b1004</a> <a href=https://www.genome.jp/entry/eco:b3713>b3713</a> <a href=https://www.genome.jp/entry/eco:b1109>b1109</a> <a href=https://www.genome.jp/entry/eco:b0046>b0046</a> <a href=https://www.genome.jp/entry/eco:b3236>b3236</a> <a href=https://www.genome.jp/entry/eco:b1779>b1779</a> <a href=https://www.genome.jp/entry/eco:b0261>b0261</a> <a href=https://www.genome.jp/entry/eco:b1602>b1602</a> <a href=https://www.genome.jp/entry/eco:b2913>b2913</a> <a href=https://www.genome.jp/entry/eco:b3915>b3915</a> <a href=https://www.genome.jp/entry/eco:b2975>b2975</a> <a href=https://www.genome.jp/entry/eco:b3603>b3603</a> <a href=https://www.genome.jp/entry/eco:b2492>b2492</a> <a href=https://www.genome.jp/entry/eco:b0904>b0904</a> <a href=https://www.genome.jp/entry/eco:b1380>b1380</a> <a href=https://www.genome.jp/entry/eco:b1695>b1695</a> <a href=https://www.genome.jp/entry/eco:b1771>b1771</a> <a href=https://www.genome.jp/entry/eco:b0606>b0606</a> <a href=https://www.genome.jp/entry/eco:b2285>b2285</a> <a href=https://www.genome.jp/entry/eco:b1009>b1009</a> <a href=https://www.genome.jp/entry/eco:b4209>b4209</a> &nbsp <a href="../alternativeGenes/alternative_genes&iML1515&glyclt_c&41&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.502727 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.478393 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 36.987550<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 5.429413<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.484934<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.126597<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.098129<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe3_e">EX_fe3_e</a> : 0.008074<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.004361<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.002617<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.002617<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000356<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000347<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000171<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ni2_e">EX_ni2_e</a> : 0.000162<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cobalt2_e">EX_cobalt2_e</a> : 0.000013<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 51.094805<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 37.823203<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 5.398408<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 0.478393<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 0.292681<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_5drib_c">DM_5drib_c</a> : 0.000113<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_4crsol_c">DM_4crsol_c</a> : 0.000112<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iML1515&glyclt_c&minimal&41.json" download="../json/iML1515&glyclt_c&minimal&41.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iJO1366.Central metabolism&tool=Builder&model=iJO1366"> Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 21-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
