<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database [1] / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iML1515&cpe160_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iML1515">iML1515</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iML1515/metabolites/cpe160_c">cpe160_c</a><br>
List of minimal gene deletion strategies <a href="../csv/iML1515&cpe160_c.csv" download="iML1515&cpe160_c.csv">(Download)</a><br><br>
Gene deletion strategy (84 of 107: <a href="iML1515&cpe160_c&85.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 26<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:b1054>b1054</a> <a href=https://www.genome.jp/entry/eco:b4382>b4382</a> <a href=https://www.genome.jp/entry/eco:b3831>b3831</a> <a href=https://www.genome.jp/entry/eco:b4384>b4384</a> <a href=https://www.genome.jp/entry/eco:b3614>b3614</a> <a href=https://www.genome.jp/entry/eco:b0910>b0910</a> <a href=https://www.genome.jp/entry/eco:b4152>b4152</a> <a href=https://www.genome.jp/entry/eco:b2781>b2781</a> <a href=https://www.genome.jp/entry/eco:b0030>b0030</a> <a href=https://www.genome.jp/entry/eco:b1612>b1612</a> <a href=https://www.genome.jp/entry/eco:b1611>b1611</a> <a href=https://www.genome.jp/entry/eco:b4122>b4122</a> <a href=https://www.genome.jp/entry/eco:b0651>b0651</a> <a href=https://www.genome.jp/entry/eco:b2162>b2162</a> <a href=https://www.genome.jp/entry/eco:b1759>b1759</a> <a href=https://www.genome.jp/entry/eco:b1701>b1701</a> <a href=https://www.genome.jp/entry/eco:b1805>b1805</a> <a href=https://www.genome.jp/entry/eco:b4138>b4138</a> <a href=https://www.genome.jp/entry/eco:b4123>b4123</a> <a href=https://www.genome.jp/entry/eco:b0621>b0621</a> <a href=https://www.genome.jp/entry/eco:b2406>b2406</a> <a href=https://www.genome.jp/entry/eco:b2197>b2197</a> <a href=https://www.genome.jp/entry/eco:b3918>b3918</a> <a href=https://www.genome.jp/entry/eco:b0789>b0789</a> <a href=https://www.genome.jp/entry/eco:b1249>b1249</a> <a href=https://www.genome.jp/entry/eco:b1206>b1206</a> &nbsp <a href="../alternativeGenes/alternative_genes&iML1515&cpe160_c&84&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.555314 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.293287 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 17.173102<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 7.078235<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.828946<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.139839<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.108393<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe2_e">EX_fe2_e</a> : 0.008919<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.004817<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.002890<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.002890<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000394<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000384<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000189<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ni2_e">EX_ni2_e</a> : 0.000179<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cobalt2_e">EX_cobalt2_e</a> : 0.000014<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 40.556040<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 21.877213<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 6.754918<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_succ_e">EX_succ_e</a> : 0.579077<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ura_e">EX_ura_e</a> : 0.393800<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 0.293287<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_5drib_c">DM_5drib_c</a> : 0.000125<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_4crsol_c">DM_4crsol_c</a> : 0.000124<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iML1515&cpe160_c&minimal&84.json" download="../json/iML1515&cpe160_c&minimal&84.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iJO1366.Central metabolism&tool=Builder&model=iJO1366"> Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 21-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
