<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Wild type metabolic flow</font></b><br><br>
<a href="../indexFiles/index_iMM904.html">Search target metabolites</a>.<br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iMM904">iMM904</a> [1].<br>
<br>
In wild type, when growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.287866 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 2.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 1.611029<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.056911<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.022252<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 18.021084<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_etoh_e">EX_etoh_e</a> : 15.815475<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 5.640455<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 1.449530<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_for_e">EX_for_e</a> : 0.001488<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iMM904&wildtype.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iMM904.Central carbon metabolism&tool=Builder&model=iMM904"> Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 27-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
