<head>
<title>MetNetComp</title>
<script async src="https://www.googletagmanager.com/gtag/js?id=G-014FPN14MZ"></script>
 <script> window.dataLayer = window.dataLayer || []; function
 gtag(){dataLayer.push(arguments);} gtag('js', new Date());
 gtag('config', 'G-014FPN14MZ');</script>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database [1] / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=STM_v1_0&ppp9_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/STM_v1_0">STM_v1_0</a> [2].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/STM_v1_0/metabolites/ppp9_c">ppp9_c</a><br>
List of minimal gene deletion strategies <a href="../csv/STM_v1_0&ppp9_c.csv" download="STM_v1_0&ppp9_c.csv">(Download)</a><br><br>
Gene deletion strategy (9 of 20: <a href="STM_v1_0&ppp9_c&10.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 25<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/stm:STM4483>STM4483</a> <a href=https://www.genome.jp/entry/stm:STM2421>STM2421</a> <a href=https://www.genome.jp/entry/stm:STM0146>STM0146</a> <a href=https://www.genome.jp/entry/stm:STM3680>STM3680</a> <a href=https://www.genome.jp/entry/stm:STM1290>STM1290</a> <a href=https://www.genome.jp/entry/stm:STM1884>STM1884</a> <a href=https://www.genome.jp/entry/stm:STM0517>STM0517</a> <a href=https://www.genome.jp/entry/stm:STM1570>STM1570</a> <a href=https://www.genome.jp/entry/stm:STM4036>STM4036</a> <a href=https://www.genome.jp/entry/stm:STM3359>STM3359</a> <a href=https://www.genome.jp/entry/stm:STM1566>STM1566</a> <a href=https://www.genome.jp/entry/stm:STM2472>STM2472</a> <a href=https://www.genome.jp/entry/stm:STM2403>STM2403</a> <a href=https://www.genome.jp/entry/stm:STM0393>STM0393</a> <a href=https://www.genome.jp/entry/stm:STM0542>STM0542</a> <a href=https://www.genome.jp/entry/stm:STM2317>STM2317</a> <a href=https://www.genome.jp/entry/stm:STM1480>STM1480</a> <a href=https://www.genome.jp/entry/stm:STM0700>STM0700</a> <a href=https://www.genome.jp/entry/stm:STM2299>STM2299</a> <a href=https://www.genome.jp/entry/stm:STM1933>STM1933</a> <a href=https://www.genome.jp/entry/stm:STM3063>STM3063</a> <a href=https://www.genome.jp/entry/stm:STM4301>STM4301</a> <a href=https://www.genome.jp/entry/stm:STM3599>STM3599</a> <a href=https://www.genome.jp/entry/stm:STM4325>STM4325</a> <a href=https://www.genome.jp/entry/stm:STM0627>STM0627</a> &nbsp <a href="../alternativeGenes/alternative_genes&STM_v1_0&ppp9_c&9&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.015755 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.001139 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 9.552803<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 5.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 0.209902<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.050265<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.002798<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.001921<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.000124<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe2_e">EX_fe2_e</a> : 0.000115<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.000075<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.000075<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000050<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000050<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mobd_e">EX_mobd_e</a> : 0.000050<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000050<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cobalt2_e">EX_cobalt2_e</a> : 0.000049<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 10.953063<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 9.575518<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 9.351499<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 7.729881<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_succ_e">EX_succ_e</a> : 0.707346<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_kdo2lipid4_e">EX_kdo2lipid4_e</a> : 0.018148<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glyclt_e">EX_glyclt_e</a> : 0.000788<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/STM_v1_0&ppp9_c&minimal&9.json" download="../json/STM_v1_0&ppp9_c&minimal&9.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/"> Escher site</a> [3].<br><br>
References<br>
[1] Tamura, T. MetNetComp: Database for minimal and maximal gene deletion strategies for growth-coupled production of genome-scale metabolic networks, IEEE/ACM Transactions on Computational Biology and Bioinformatics, in press.<br>
[2] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 27-Sep-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
